statsmodels/tsa/vector_ar/tests/test_vecm.py

```diff
@@ -1337,6 +1337,7 @@ def test_exceptions():
                         vecm_res.predict, 5, None, exog_coint_fc)  # passed as exog_fc-argument!
 
 
+@pytest.mark.smoke
 def test_select_coint_rank():  # This is only a smoke test.
```

Comment:
another one that uses "smoke test" to indicate that the numbers are not tested directly, or against a verified source.
It still assert the inequalities